jaxrs/src/test/java/io/airlift/jaxrs/TestLegacyUriMode.java

```diff
@@ -0,0 +1,74 @@
+package io.airlift.jaxrs;
+
+import com.google.common.collect.ImmutableList;
+import com.google.inject.BindingAnnotation;
+import com.google.inject.Injector;
+import com.google.inject.Key;
+import com.google.inject.Module;
+import io.airlift.bootstrap.Bootstrap;
+import io.airlift.http.client.HttpClient;
+import io.airlift.http.client.Request;
+import io.airlift.http.server.HttpServerInfo;
+import io.airlift.http.server.testing.TestingHttpServerModule;
+import io.airlift.json.JsonModule;
+import io.airlift.node.testing.TestingNodeModule;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.PathParam;
+import org.testng.annotations.Test;
+
+import java.lang.annotation.Retention;
+import java.net.URI;
+import java.util.List;
+
+import static io.airlift.http.client.HttpClientBinder.httpClientBinder;
+import static io.airlift.http.client.JsonResponseHandler.createJsonResponseHandler;
+import static io.airlift.http.client.Request.Builder.prepareGet;
+import static io.airlift.jaxrs.JaxrsBinder.jaxrsBinder;
+import static io.airlift.json.JsonCodec.listJsonCodec;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+import static org.testng.Assert.assertEquals;
+
+public class TestLegacyUriMode
+{
+    @Path("/legacy")
+    public static class MyResource
+    {
+        @GET
+        @Path("{a:.*}/{b:.*}/{c:.*}")
```

Comment:
Isn't this normally done as
```java
@Path("{a}/{b}/{c}")

Reply:
I wasn't sure how it was used. I can try both.